tests/trac/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2025 Frank Sommer <Frank.Sommer@sherpa-software.de>
+#
+# Licensed under GNU Affero General Public License v3 or later (AGPLv3+)
+# https://www.gnu.org/licenses/agpl-3.0.html
+
+FROM python:3.11-slim
```

Comment:
Please add this directory to https://github.com/kiwitcms/trackers-integration/blob/main/.github/dependabot.yml